slurp/src/main/webapp/index.html

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
-    <title>Slurp</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Woah, I have never seen this tag before. Pretty cool stuff.

Reply:
It's a good thing to have for mobile devices and other small screens! Won't take care of everything but is usually a good first step.